pkg/commands/patch/patch_manager.go

```diff
@@ -192,21 +195,15 @@ func (p *PatchManager) RenderPatchForFile(filename string, plain bool, reverse b
 
 func (p *PatchManager) renderEachFilePatch(plain bool) []string {
 	// sort files by name then iterate through and render each patch
-	filenames := make([]string, len(p.fileInfoMap))
-	index := 0
-	for filename := range p.fileInfoMap {
-		filenames[index] = filename
-		index++
-	}
+	filenames := maps.Keys(p.fileInfoMap)
```

Comment:
huge improvement here both in line count and readability. It took me a while to work out why there was a standalone index variable in the original code because it hadn't occurred to me that we were ranging over a map